src/common-components/PasswordField.jsx

```diff
@@ -21,6 +21,13 @@ const PasswordField = (props) => {
     setShowTooltip(props.showRequirements && false);
   };
 
+  const handleFocus = (e) => {
+    if (props.handleFocus) {
+      props.handleFocus(e);
+      setTimeout(() => setShowTooltip(props.showRequirements && true), 150);
+    }
+  };
```

Comment:
setTimeout should be called even if handleFocus is not present
```suggestion
  const handleFocus = (e) => {
    if (props.handleFocus) {
      props.handleFocus(e);
    }
    setTimeout(() => setShowTooltip(props.showRequirements && true), 150);
  };
```